packages/mongodb-chatbot-server/src/processors/makeVerifiedAnswerGenerateUserPrompt.ts

```diff
@@ -0,0 +1,44 @@
+import { FindVerifiedAnswerFunc } from "./FindVerifiedAnswerFunc";
+import {
+  GenerateUserPromptFunc,
+  GenerateUserPromptFuncReturnValue,
+} from "./GenerateUserPromptFunc";
+
+export interface MakeVerifiedAnswerGenerateUserPromptParams {
+  /**
+    Find content based on the user's message and preprocessing.
+   */
+  findVerifiedAnswer: FindVerifiedAnswerFunc;
+
+  continuation: GenerateUserPromptFunc;
```

Comment:
I think this could do with a doc comment at minimum. Moreover, I wonder if `continuation` is a bit too jargon-y. A more descriptive name could be helpful, e.g.:

- `onNoVerifiedAnswerFound`
- `handleNoVerifiedAnswerFound`

Reply:
> handleNoVerifiedAnswerFound

i like this one personally. and +1 to doc comment